lemminx-liberty/src/test/java/io/openliberty/LibertyDiagnosticTest.java

```diff
@@ -464,6 +464,25 @@ public void testInvalidPlatformDiagnostic() throws BadLocationException {
 
         XMLAssert.testDiagnosticsFor(serverXML, null, null, serverXMLURI,
                 invalid1, invalid2, invalid3, invalid4);
+        //expecting code action to show all javaee platforms
+        List<String> featuresStartWithJavaEE = new ArrayList<>();
+        featuresStartWithJavaEE.add("javaee-6.0");
+        featuresStartWithJavaEE.add("javaee-7.0");
+        featuresStartWithJavaEE.add("javaee-8.0");
```

Comment:
Maybe the code needs to change more for the quick fix. Why are we suggesting platforms that they already included (javaee-7.0, javaee-8.0)?

Reply:
See the quick fix for ReplaceFeature method `getFeatureReplacements` which checks for existing features.

        // get list of existing features to exclude from list of possible replacements
        // also exclude any feature with a different version that matches an existing feature

Reply:
Done